src/explore-education-statistics-admin/package.json

```diff
@@ -53,6 +54,7 @@
     "cross-fetch": "^3.0.4",
     "css-loader": "^3.5.1",
     "date-fns": "^2.11.1",
+    "dompurify": "^2.0.8",
```

Comment:
If we abstract DOMPurify into a common util function, we only need to install this in common.